sorting/CombSort.R

```diff
@@ -0,0 +1,27 @@
+# Comb sort in R:
+
+comb.sort <- function(elements.vec) {
+  a = elements.vec
```

Comment:
Looks okay overall, but it would be nice to just have `elements.vec` in place of instances of `a`. (discarding the use of an extra variable)